package.json

```diff
@@ -25,6 +25,7 @@
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
     "expo": "~51.0.28",
+    "expo-av": "^14.0.7",
```

Comment:
what is this for?

Reply:
This was for handling audio in expo. We can remove it for now.